docs/versioned/spark/spark-writes.md

```diff
@@ -98,6 +98,11 @@ WHEN NOT MATCHED AND s.event_time > still_valid_threshold THEN INSERT (id, count
 
 Only one record in the source data can update any given row of the target table, or else an error will be thrown.
 
+!!! Note
```

Comment:
Does the three !!! renders anything?  (I dont see it when viewing the file as md).  Should we just use a normal sub-header here?

Reply:
It causes this to render in a text box. I would prefer having a full section on the copy-on-write vs merge-on-read distinction and then refer to it from the individual command sections.

Reply:
we have a PR for that already,
https://github.com/apache/iceberg/pull/3432